tests/e2e/tests/regression/cli/cli.e2e.ts

```diff
@@ -91,3 +101,81 @@ test
         //Verify result
         await t.expect(cliPage.cliOutputResponseSuccess.count).eql(repeats, `CLI contains ${repeats} results`);
 });
+test
+    .meta({ rte: rte.standalone })
+    .before(async () => {
+        await acceptLicenseTerms();
+    })
```

Comment:
FYI This overrides the beforeEach fixture hook

Reply:
in this hook the db is not added, because it should be added inside test 
and in the beforeEach hook the standalone db without any index is added